src/sections/General/Navigation/utility/ScrollspyMenu.js

```diff
@@ -16,13 +16,16 @@ import SupportingArrows from "../../../../sections/Meshmap/Meshmap-collaborate/i
 import EmptyLight from "../../../../sections/Meshmap/Meshmap-collaborate/images/banner-transitions/empty-light.svg";
 import EmptyDark from "../../../../sections/Meshmap/Meshmap-collaborate/images/banner-transitions/empty-dark.svg";
 import { useInView } from "react-intersection-observer";
+import DropDownIcon from "./icons/dropDownIcon.svg";
```

Comment:
We are using react-icons library, so please use import an icon from there.
Use this `IoMdArrowDropdown`, and import it like
`import { IoMdArrowDropdown } from "@react-icons/all-files/io/IoMdArrowDropdown";`